File: 2160. Minimum Sum of Four Digit Number After Splitting Digits.py
class Solution:
    def minimumSum(self, num: int) -> int:
        str_num = str(num)
        sorted_num = sorted(str_num)
        
        num1 = ''
        num2 = ''

        num1 += sorted_num[0]
        num2 += sorted_num[1]
        num1 += sorted_num[2]
        num2 += sorted_num[3]

        print(f'{num1} {num2}')

        return int(num1) + int(num2)

File: 2160. Minimum Sum of Four Digit Number After Splitting Digits.cs
public class Solution {
    public int MinimumSum(int num) {
        string stringNum = num.ToString();
        char[] charArray = stringNum.ToCharArray();
        Array.Sort(charArray);
        string sortedStringNum = new string(charArray);
        string num1 = $"{sortedStringNum[0]}{sortedStringNum[2]}";
        string num2 = $"{sortedStringNum[1]}{sortedStringNum[3]}";

        return int.Parse(num1) + int.Parse(num2);
    }
}
